io/rtp/src/main/java/org/mobicents/media/server/impl/rtp/channels/MediaChannel.java

```diff
@@ -644,8 +644,27 @@ public void negotiateFormats(MediaDescriptionField media) {
 		
 		// Map payload types tp RTP Format
 		for (int payloadType : media.getPayloadTypes()) {
-			RTPFormat format = AVProfile.getFormat(payloadType, AVProfile.AUDIO);
+			RTPFormat format = null;
+			if (payloadType < 96 )
```

Comment:
Magic number. Create a constant for this value with proper comment explaining it.